docs/developer.md

```diff
@@ -131,9 +140,21 @@ $ cd $GOPATH/src/github.com/openebs/maya
 
 ### Make your changes and build them
 
+To build
+To build just standalone CLI for development:
+
  ```sh
- cd $GOPATH/src/github.com/openebs/maya
- make all
+
+ cd path/to/directory/openebs/maya
+ GO111MODULE=on make mayactl
```

Comment:
I think we can remove this. `go` will use go-module if go.mod file exists in root package.

Reply:
true, it does seem a little redundant :) will fix it